src/makeRobot.js

```diff
@@ -39,6 +39,70 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward: function(value = 1) {
+      if (value <= 0) {
+        return this;
+      }
+
+      this.coords.y += value;
+
+      return this;
```

Comment:
Так не будеш повторюватися
```suggestion
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
```